File: src/main/java/no/nav/boot/conditionals/OnLocalOrTestCondition.java
package no.nav.boot.conditionals;

import static no.nav.boot.conditionals.Cluster.localOrTest;

import org.springframework.core.type.AnnotatedTypeMetadata;

public class OnLocalOrTestCondition extends OnClusterCondition {

    @Override
    protected Cluster[] clusters(AnnotatedTypeMetadata md) {
        return localOrTest();
    }
}

File: src/main/kotlin/no/nav/boot/conditionals/OnLocalOrTestCondition.kt
package no.nav.boot.conditionals

import no.nav.boot.conditionals.Cluster.Companion.localOrTest
import org.springframework.core.type.AnnotatedTypeMetadata

class OnLocalOrTestCondition : OnClusterCondition() {
    override fun clusters(md: AnnotatedTypeMetadata) = localOrTest()
}